xbmc/interfaces/builtins/AddonBuiltins.cpp

```diff
@@ -226,7 +226,7 @@ static int RunScript(const std::vector<std::string>& params)
 #if defined(TARGET_DARWIN_OSX)
   std::string execute;
   std::string parameter = params.size() ? params[0] : "";
-  if (URIUtils::HasExtension(parameter, ".applescript|.scpt"))
+  if (URIUtils::HasExtension(parameter, {".applescript", ".scpt"}))
```

Comment:
It looks like `static` vectors have a small speedup over stack vectors, should this be converted?